ui/src/forms/matchers.ts

```diff
@@ -0,0 +1,19 @@
+import { cc } from '@cube-creator/core/namespace'
+import type { SingleEditor } from '@hydrofoil/shaperone-core'
+import { editor } from '@/forms/bulma'
+
+export const projectKindSelector: SingleEditor = {
```

Comment:
Why is it specific to `projectKind`? Having code that is domain-specific in the matchers will be very confusing. Any way we can hard-set the component to use directly in the shape?

Reply:
I guess initially I want an example of a very specific matcher but radio buttons hardly are unique UI component, I agree 👌